### ISSUE TITLE
release --disable-openapi-validation

### ISSUE THREAD

**Issue by iameli** [databus23/helm-diff#220]
EDIT: Whoops, this already exists. Could we get a 3.1.2 release that includes the fix? Thanks!

Love the plugin, thanks so much for writing it!

I'm running against a cluster with the [ServiceTopology](https://kubernetes.io/docs/concepts/services-networking/service-topology/) feature gate enabled. Helm doesn't know about it, so it fails OpenAPI validation.
```
> helm diff upgrade -f .build/staging-livepeer.rkv.values.yaml staging-livepeer ./charts/livepeer
Error: Failed to render chart: exit status 1: coalesce.go:199: warning: destination for annotations is a table. Ignoring non-table value []
coalesce.go:199: warning: destination for annotations is a table. Ignoring non-table value []
coalesce.go:199: warning: destination for annotations is a table. Ignoring non-table value []
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Service.spec): unknown field "topologyKey" in io.k8s.api.core.v1.ServiceSpec

Error: plugin "diff" exited with error
```

With the regular `helm upgrade` command, I can bypass validation and still upgrade my cluster using `helm upgrade --disable-openapi-validation`. It would be nice if I could use `helm-diff` in the same way.

**Comment by iameli** [comment]
Oh, hilarious, this is literally the tip of master right now. My bad! https://github.com/databus23/helm-diff/commit/26d61fa2aad8476d40307f662db5edd59b07b79b

**Comment by iameli** [comment]
(Will close when this releases)

**Comment by iameli** [comment]
(In the meantime I threw it up on a fork so `helm plugin install https://github.com/livepeer/helm-diff --version master` works)

**Comment by mumoshu** [comment]
@iameli Finally this has been released in v3.1.2. Thanks for patience!

JFYI, I had to use `--disable-validation` added in #228 for "installing" prometheus-operator because it needs to "ignore" validation errors due to missing(unregistered) API versions.

For your original issue, `--disable-openapi-validation` should indeed be enough.
